docs/buildah-add.md

```diff
@@ -47,5 +47,52 @@ buildah add containerID 'https://github.com/containers/buildah/blob/master/READM
 
 buildah add containerID 'passwd' 'certs.d' /etc
 
+## FILES
+
+### `.dockerignore`
+
+If the file .dockerignore exists in the context directory, `buildah copy` reads 
+its contents. Buildah uses the content to exclude files and directories from 
+the context directory, when copying content into the image.
+
+Users can specify a series of Unix shell globas in a .dockerignore file to
+identify files to exclude. 
+
+Buildah supports a special wildcard string `**` which matches any number of
+directories (including zero). For example, **/*.go will exclude all files that
+end with .go that are found in all directories.
+
+Example .dockerignore file:
+
+```
+# comment
+*/temp*
+**/temp*
+temp?
+```
+
+`*/temp*`
+Excludes files and directories whose names start with temp in any immediate subdirectory of the root. For example, the plain file /somedir/temporary.txt is excluded, as is the directory /somedir/temp.
+
+`**/temp*`
+Excludes files and directories starting with temp from any subdirectory. 
+
+`temp?`
+Excludes files and directories in the root directory whose names are a one-character extension of temp. For example, /tempa and /tempb are excluded.
+
+Lines starting with ! (exclamation mark) can be used to make exceptions to
+exclusions. The following is an example .dockerignore file that uses this
+mechanism:
+```
+*.md
+!README.md
+```
+
+All markdown files except README.md are excluded from the context.
+
+For more information see:
+
+https://docs.docker.com/engine/reference/builder/#dockerignore-file
```

Comment:
hate, hate, hate having a Docker ref in our man page....

Reply:
Yes, but this is the best reference I could find, and I did not want to just cut and paste the content into out man page.

Reply:
@TomSweeneyRedHat, do you have an alternative?

I think it's okay to refer to Docker docs here. Everything Dockerfile-related is effectively owned by Docker as there's no build spec.